src/WebView.android.tsx

```diff
@@ -20,13 +20,23 @@ import {
   useWebViewLogic,
 } from './WebViewShared';
 import {
-  AndroidWebViewProps, WebViewSourceUri,
+  AndroidWebViewProps,
+  WebViewSourceUri,
+  type WebViewMessageEvent,
+  type ShouldStartLoadRequestEvent,
 } from './WebViewTypes';
 
 import styles from './WebView.styles';
 
 const { resolveAssetSource } = Image;
 
+const registerCallableModule: (name: string, module: Object) => void
+  // `registerCallableModule()` is available in React Native 0.74 and above.
+  // Fallback to use `BatchedBridge.registerCallableModule()` for older versions.
+  // eslint-disable-next-line global-require
+  = require('react-native').registerCallableModule
+  ?? BatchedBridge.registerCallableModule.bind(BatchedBridge);
```

Comment:
@Kudo, in 0.73, we also have access to global.RN$registerCallableModule in bridgeless mode. If you also include that here, then we could make this library bridgeless mode compatible in 0.73 as well.

Reply:
thanks @RSNara for taking a look and having the feedback. i'm lean toward to claim the bridgeless support for react-native-webview from 0.74, because i didn't run any bridgeless test on on 0.73.